components/content_settings/core/browser/brave_content_settings_pref_provider_unittest.cc

```diff
@@ -611,4 +612,25 @@ TEST_F(BravePrefProviderTest, TestShieldsSettingsMigrationV2toV3) {
   provider.ShutdownOnUIThread();
 }
 
+TEST_F(BravePrefProviderTest, EnsureNoWildcardEntries) {
+  BravePrefProvider provider(
+      testing_profile()->GetPrefs(), false /* incognito */,
+      true /* store_last_modified */, false /* restore_session */);
+  ShieldsEnabledSetting shields_enabled_settings(&provider);
+  GURL example_url("https://example.com");
+  shields_enabled_settings.CheckSettingsAreDefault(example_url);
+  // Set wildcard entry
+  auto pattern = ContentSettingsPattern::Wildcard();
+  provider.SetWebsiteSetting(pattern, pattern,
+                             ContentSettingsType::BRAVE_SHIELDS,
+                             base::Value(CONTENT_SETTING_ALLOW), {});
+  // Verify global has changed
```

Comment:
I guess these are TODOs?

Reply:
No it's a comment for what the next line is attempting to achieve

Reply:
ah, I see now